playground/AzureAIFoundryEndToEnd/AzureAIFoundryEndToEnd.AppHost/Program.cs

```diff
@@ -0,0 +1,28 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+var builder = DistributedApplication.CreateBuilder(args);
+
+var foundry = builder.AddAzureAIFoundry("foundry")
+    .RunAsFoundryLocal()
```

Comment:
When I try to F5 this app, I'm getting an error that says my Azure provisioning information is missing.

We shouldn't require Azure provisioning info when the app is running as local.

Reply:
Sounds good

Reply:
https://github.com/dotnet/aspire/pull/9974/commits/050562992e9b24f8c1518a665ce648f1662c4bcb

Borderline would require a distinct PR. Found `ICutomResource` but needed an annotation in this case. I looked for all usages of `IsContainer()` in the context of "not deployable" resource.